.NET/Microsoft.Recognizers.Definitions.Common/Japanese/DateTimeDefinitions.cs

```diff
@@ -166,28 +166,28 @@ public static class DateTimeDefinitions
       public const string SetEachPrefixRegex = @"(?<each>(每)\s*$)";
       public const string SetLastRegex = @"(?<last>last|this|next)";
       public const string SetEachDayRegex = @"(每|每一)(天|日)\s*$";
-      public const string TimeHourNumRegex = @"(00|01|02|03|04|05|06|07|08|09|0|10|11|12|13|14|15|16|17|18|19|20|21|22|23|24|1|2|3|4|5|6|7|8|9)";
-      public const string TimeMinuteNumRegex = @"(00|01|02|03|04|05|06|07|08|09|10|11|12|13|14|15|16|17|18|19|20|21|22|23|24|25|26|27|28|29|30|31|32|33|34|35|36|37|38|39|40|41|42|43|44|45|46|47|48|49|50|51|52|53|54|55|56|57|58|59|0|1|2|3|4|5|6|7|8|9)";
-      public const string TimeSecondNumRegex = @"(00|01|02|03|04|05|06|07|08|09|10|11|12|13|14|15|16|17|18|19|20|21|22|23|24|25|26|27|28|29|30|31|32|33|34|35|36|37|38|39|40|41|42|43|44|45|46|47|48|49|50|51|52|53|54|55|56|57|58|59|0|1|2|3|4|5|6|7|8|9)";
-      public const string TimeHourCJKRegex = @"([零〇一二两三四五六七八九]|二十[一二三四]?|十[一二三四五六七八九]?)";
+      public const string TimeHourNumRegex = @"(00|01|02|03|04|05|06|07|08|09|0|1|2|3|4|5|6|7|8|9|10|11|12|13|14|15|16|17|18|19|20|21|22|23|24|08|09|００|０１|０２|０３|０４|０５|０６|０７|０８|０９|０|１|２|３|４|５|６|７|８|９|１０|１１|１２|１３|１４|１５|１６|１７|１８|１９|２０|２１|２２|２３|２４)";

```

Comment:
There should be no need to add the different width chars (e.g. "００"). This is normalized before hitting the regexes.

Reply:
This can also be simplified to using \d and [] groups.

Reply:
right, will do

Reply:
done